File: Movil/mobile/src/main/java/com/example/movil/models/UsuarioModel.kt
package com.example.movil.models

class UsuarioModel(
    val usuarioId: Int,
    val cuenta: String,
    val contrasena: String,
    val reiniciarContrasena: Byte,
    val terminos: Byte,
    val activo: Byte,
    val personaId: Int,
    val persona: Persona,
    val tipoUsuarioId: Int,
    val tipoUsuario: TipoUsuario?,
    val perfilId: Int
)

File: Servicios/Models/ViewModel/UsuarioModel.cs
using Servicios.Models.Tables;

namespace Servicios.Models.ViewModel
{
    public class UsuarioModel
    {
        public int usuarioId { get; set; }
        public string cuenta { get; set; }
        public string contrasena { get; set; }
        public byte reiniciarContrasena { get; set; }
        public byte terminos { get; set; }
        public byte activo { get; set; }
        public int personaId { get; set; }
        public int PerfilId { get; set; }
        public Persona persona { get; set; }
        public int tipoUsuarioId { get; set; }
        public TipoUsuario tipoUsuario { get; set; }
    }
}
